mico-admin/src/app/service-detail-overview/service-detail-overview.component.ts

```diff
@@ -117,36 +169,48 @@ export class ServiceDetailOverviewComponent implements OnInit, OnDestroy {
         });
     }
 
-    addInternalDependency() {
+    /**
+     * action triggered in ui
+     */
+    addDependee() {
         const dialogRef = this.dialog.open(ServicePickerComponent, {
             data: {
                 filter: 'internal',
                 choice: 'multi',
-                exisitingDependencies: this.internalDependencies,
-                serviceId: this.id
+                exisitingDependencies: this.dependees,
+                serviceId: this.shortName,
             }
         });
-        this.subInternalDependency = dialogRef.afterClosed().subscribe(result => {
+        this.subDependeesDialog = dialogRef.afterClosed().subscribe(result => {
             console.log(result);
             // TODO use result in a useful way
         });
     }
 
+    /**
+     * action triggered in ui
+     */
     addExternalDependency() {
+        // TODO is this method still relevant? There are no internal/external dependencies in the backend.
+        // So the 'External' list was changed to dependers.
```

Comment:
dependers and dependees are about services that depend on this service or services this  service depends on! Not if the service is external or not. Because only dependencys of the current service should be editable here one is only for display.